src/components/NcHeaderMenu/NcHeaderMenu.vue

```diff
@@ -363,10 +364,6 @@ $externalMargin: 8px;
 		opacity: 1;
 	}
 
-	&__trigger:focus-visible {
```

Comment:
I think this needs to stay, see @Pytal comments

Reply:
I tried leaving this in to take out the border outline, but since the NcButton has the `!important` styling precedent on its outline and bordering styling, I had to use :deep styling in the newest PR.

`NcButton`
![image](https://github.com/nextcloud-libraries/nextcloud-vue/assets/110193237/1255bb29-6d2a-4221-a346-02eab9f7f486)

And this did not work :(
![image](https://github.com/nextcloud-libraries/nextcloud-vue/assets/110193237/2b9a2fb0-63f0-4aab-b6c8-97ba5959bbf8)

Reply:
yes because both have the same specificity, you also can just increase it, currently the style is just on `.header-menu__trigger`, but it should work with `.header-menu .header-menu__trigger:focus-visible`.

Something like:
```scss
.header-menu {
    // ...
    #{&}__trigger:focus-visible {
        // here
    }
}
```

Reply:
not sure if it is better just an option ;)

Reply:
Ahhh, only now do I get what you mean! I changed it to how you requested, actually, because I am going to fix this problem on the unified search button right now, and for code styling being consistant, I will use your solution (plus I try not to use `:deep` as much as possible)